README.md

```diff
@@ -5,3 +5,12 @@ Create Gen3 apps without configuration.
 Clone this repository and then run `npm run initialize {my-repo-name}.` 
 In the directory above this repository, there will be a boilerplate app created with the name `{my-repo-name}`.
 It will automatically run, and can be run with `npm start`.
```

Comment:
Can you just remove this `it will automatically run` since `npm start` is gone now?